python/amici/petab_objective.py

```diff
@@ -153,9 +153,10 @@ def simulate_petab(
 
     # Log results
     sim_cond = petab_problem.get_simulation_conditions_from_measurement_df()
-    for i, rdata in enumerate(rdatas):
-        logger.debug(f"Condition: {sim_cond.iloc[i, :].values}, status: "
-                     f"{rdata['status']}, llh: {rdata['llh']}")
+    if not sim_cond.empty:
+        for i, rdata in enumerate(rdatas):
+            logger.debug(f"Condition: {sim_cond.iloc[i, :].values}, status: "
+                         f"{rdata['status']}, llh: {rdata['llh']}")
```

Comment:
Might be confusing to look at the log and not seeing any output for the respective condition? Maybe just replace `{sim_cond.iloc[i, :].values}` by `N/A` if unavailable?